defog_data/advising/advising.sql

```diff
@@ -267,5 +267,14 @@ INSERT INTO public.student_record (student_id, course_id, semester, grade, how,
 (3, 4, 2, 'B+', 'in-person', NULL, 'Yes', NULL, 1, 4)
 ;
 
-
+INSERT INTO public.gsi (course_offering_id, student_id) VALUES
```

Comment:
Another alternative is to use a query if we already know a deterministic relationship between this and other tables:
```sql
INSERT INTO public.gsi (course_offering_id, student_id)
SELECT cr.offering_id, sr.student_id
FROM public.student_record sr
JOIN public.course_offering cr ON sr.offering_id = cr.offering_id AND sr.course_id = cr.course_id AND sr.semester = cr.semester
ORDER BY sr.student_id, cr.offering_id;
```